CHANGELOG.md

```diff
@@ -28,7 +28,8 @@ release.
 </tr>
 <tr>
   <td valign="top">
-    <b><a href="doc/changelogs/CHANGELOG_V7.md#7.5.0">7.5.0</a></b><br/>
+    <b><a href="doc/changelogs/CHANGELOG_V7.md#7.6.0">7.6.0</a></b><br/>
+    <a href="doc/changelogs/CHANGELOG_V7.md#7.4.0">7.5.0</a><br/>
```

Comment:
If I’ve understood @italoacasas correctly, he’ll do the release today, with the typo fixed. I don’t think anybody actually talked about delaying it? :smile: